src/popup.html

```diff
@@ -13,6 +13,11 @@
         <img src="../images/icon48.png" width="48" height="48" class="me-3" />
         <h4>Key Retriever</h4>
       </div>
+      <div id="outdatedVersion" class="alert alert-danger d-flex align-items-center mt-1 mb-0 display-none" role="alert" style="font-size: 12px;">
```

Comment:
Let's use our flex class instead of bootstrap one that messes with display-none class (they use !important 😅 which is not very suitable for us)
```suggestion
      <div id="outdatedVersion" class="alert alert-danger flex align-items-center mt-1 mb-0 display-none" role="alert" style="font-size: 12px;">
```